apidocs/README.md

```diff
@@ -0,0 +1,27 @@
+# API Documentation
+
+The purpose of this sub-project/folder is to generate an html file
+to be consumed by the documentation site. If you want to view the
+file generated for this, perform this step:
+
+```sh-session
+$ cd "$(git rev-parse --show-toplevel)" # cd to project root
+$ docker-compose run --rm apidocs > api.html
+
+This will generate the file into the project root.
+
+```
+
+## Working on the API docs
+
+If you're working on the API docs locally, you can get a live-updating
+dev server:
+
+```sh-session
+$ cd $(git rev-parse --show-cdup) # start in the project root
+$ docker-compose run --rm --service-ports apidocs -w
+```
+
+Then `open localhost:3000` to see the API docs in your browser.
```

Comment:
It feels like this line doesn't make a lot of sense here as the result of the command `docker-compose run --rm --service-ports apidocs -w` runs until the user exits.  How about changing it to something like:
```
Then, navigate to http://localhost:3000 to see the API docs in your browser.
```

Reply:
Good catch!  That was a holdover from the old docs.  Fixed!